controllers/attester_controller.go

```diff
@@ -36,9 +37,10 @@ import (
 // AttesterReconciler reconciles a Attester object
 type AttesterReconciler struct {
 	client.Client
-	Log       logr.Logger
-	Scheme    *runtime.Scheme
-	Attesters map[string]attester.Attester
+	Log          logr.Logger
+	Scheme       *runtime.Scheme
+	Attesters    map[string]attester.Attester
+	EventManager eventmanager.EventManager
```

Comment:
Some plumbing to make the event manager instance available to the attester, but we're weren't sure if there was a better approach here

Reply:
Yep I think that make sense and is consistent with the Enforcer Reconciler